modules/ROOT/pages/execution-engine.adoc

```diff
@@ -29,6 +29,10 @@ For connectors created with the Mule SDK, the SDK determines the most
 appropriate processing type based on how the connector is implemented. For
 details on that mechanism, refer to the xref:1.1@mule-sdk::index.adoc[Mule SDK documentation].
 
+== Optimizations
+
+There are certain conditions on flow executions where the runtime may ignore the proposed Processing Type for event processor. This is due internal optimizations. Also xref:transaction-management.adoc[Transaction Management] may affect the Proccesing Type selection.
```

Comment:
Is confusing adding "TX Management" as part of optimization, because is not a optimization, but a change in behaviour required for the correct execution of TXs